application.go

```
@@ -92,6 +106,11 @@ func (a *Application) Run() error {
 	glfw.DefaultWindowHints()
 	defer a.window.Destroy()
 
+	a.resourceWindow, err = createResourceWindow(a.window)
```

This error is never checked?